README.md

```diff
@@ -1,36 +1,84 @@
-# <img src="assets/svg/safe-logo-white.svg" height="60" valign="middle" alt="Safe{Docs}" />
+# Safe Documentation
 
 [![License](https://img.shields.io/github/license/safe-global/safe-docs)](https://github.com/safe-global/safe-docs/blob/main/LICENSE.md)
 ![GitHub package.json version (branch)](https://img.shields.io/github/package-json/v/safe-global/safe-docs)
-<!-- ![Tests](https://img.shields.io/github/actions/workflow/status/safe-global/safe-docs/test.yml?branch=main&label=tests) -->
 
-# Safe{Docs}
+This repository hosts [Safe](https://safe.global) documentation.
 
-This repository hosts [Safe](https://safe.global)'s documentation website, built with [Nextra](https://nextra.site). Live version can be found at [docs.safe.global](https://docs.safe.global).
+The project is built using [Nextra](https://nextra.site) framework and it can be found live at [docs.safe.global](https://docs.safe.global).
 
-## Quick Start
+## Installation
 
-First, run `pnpm i` to install the dependencies.
+Install the project:
 
-Then, run `pnpm dev` to start the development server and visit `localhost:3000``.
+```
+pnpm install
+```
 
-## Build
+## Development
 
-Run `pnpm build` to build the site. The output will be in the `out` folder.
+Git hooks are set up to run tests and linting checks before every `git push`. These hooks can be executed locally by running the following command:
 
-Run `pnpm start` to serve the site locally.
+```
+pnpm prepush
+```
 
-## Test
+All links in the documentation are checked for validity on every pull request. These checks can be executed locally by running the following command:
 
-Run `pnpm test` to run the tests.
+```
+pnpm linkcheck
+```
+
+## Execution
+
+The project can be run with a server that's executed in development and production mode.
+
+### Development mode
+
+Run the server in development mode:
+
+```
+pnpm dev
+```
+
+### Production mode
+
+Build the project:
+
+```
+pnpm build
+```
+
+Run de server in production mode:
+
+```
+pnpm start
+```
+
+## Testing
+
+Create an environment file in the root of the project and copy the content from the `.env.example` file:
+
+```
+touch .env
+```
+
+```
```

Comment:
The following code snippet can go and instead we can add a note reminding them to update the environment variables.